### ISSUE TITLE
Clarification about parameters recursion_depth and r

### ISSUE THREAD

**Issue by expectopatronum** [nimarb/pytorch_influence_functions#2]
Hi,
thanks for this great package!
I am wondering how to set some of the parameters. In the README you mention: 

> Combined, the original paper suggests that recursion_depth * r should equal the training dataset size, thus the above values of r = 10 and recursion_depth = 5000 are valid for CIFAR-10 with a training dataset size of 50000 items. 

Can you point me to the paragraph in the paper?

I could only find 

> We pick t to be large enough such that H̃ t stabilizes, and to reduce variance we repeat this procedure r times and average results. Empirically, we found this significantly faster than CG.

which does not tell me that recursion_depth * r should equal the training dataset size.

Thanks and best regards,
Verena

**Comment by nimarb** [comment]
Hi Verena,
Thanks for letting me know. I just checked again in the paper, and I wrong about what I wrote there. The part I was referring to was in section 4.1 where `r = 10` and `t = 5000` was used for the MNIST dataset which of course has 60k training images.

> in fact,  we  accurately estimated H−1v without even looking at every datapoint, since n= 55,000> rt. 

I guess the numbers should be considered as a guideline, as the paper also says that even `r = 1` works but that results will contain more noise. I'll change that part in the docs :)
